Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -733,6 +818,9 @@ theorem isSuccPrelimit_zero : IsSuccPrelimit (0 : Cardinal) :=
 @[deprecated isSuccPrelimit_zero (since := "2024-09-05")]
 alias isSuccLimit_zero := isSuccPrelimit_zero
 
+/-! ### Indexed cardinal `sum` -/
+
+
```

Comment:
```suggestion
```

Reply:
I thought two spaces after each section was the new convention...

Reply:
That's just what mathport did to our files but I don't think anyone is following that convention